app/views/update_username/index.html.haml

```diff
@@ -0,0 +1,6 @@
+.container
+  %h2= 'Enter Username to update' 
```

Comment:
All the text should be done via the i18n system. I think the messages can be improved as well:

"Update a changed username" is a better header. The input label can be just "Username", and the submit label can be "Update". The submit link should be styled as a button.

I think a fuller explanation right above the form would also be useful: "If a user has changed the username for their Wikimedia account, enter it here to update the username on the Dashboard."

Will this also work if you enter the new username, or only if you enter the old one?

Reply:
Accordint to the code, you've to enter your updated Wikimedia username, then it searches for the global_id associated with it, then it updates your username correspond to that global_id in DB.

Reply:
Okay, so it should explicitly say "New username", and the explanation should say "If a user has changed the username for their Wikimedia account, enter the new username here to update it on the Dashboard."